### PR TITLE
Fork changes

### PR DESCRIPTION
**Pull request by MikeAlOwe-CodeSignal**

### CODE REVIEW

**Comment by menewman** [comment]
Not related to this PR, but just curious -- did you close https://github.com/samdenty/console-feed/pull/104 on your own? Or was it the maintainer?

File: src/Component/react-inspector/index.tsx [hunk 1]
```diff
@@ -103,7 +193,12 @@ class CustomInspector extends React.PureComponent<Props, any> {
     // Root
     if (depth === 0) {
       const customNode = this.getCustomNode(data)
-      return customNode || <ObjectRootLabel name={name} data={data} />
+      return customNode || <CustomObjectRootLabel name={name} data={data} />
+    }
+
+    if (typeof data === 'string' && data.indexOf(REMAINING_KEY) !== -1) {
```
Comment (menewman) [inline src/Component/react-inspector/index.tsx:199]:
```suggestion
    if (typeof data === 'string' && data.includes(REMAINING_KEY)) {
```
[trivial, style]

**Review by menewman — APPROVED** [review]
Haha, I was too optimistic about my ability to understand this repo without delving deeper. 😄 

You're right -- the changes to add extra fields feel a little hacky, but as long as we can strip them on deserialization I suppose it's okay.

It would be great to tweak the test cases to capture that the new logic is working correctly with unit tests if possible, since the logic is complex and it can be hard to follow and feel confident in. Approving, though 👍

**Comment by MikeAlOwe-CodeSignal** [comment]
I deleted the fork from my GitHub after creating the CodeSignal fork and I think that's what caused the PR to close... I will reopen it.

**Comment by MikeAlOwe-CodeSignal** [comment]
Reopened PR on original library: https://github.com/samdenty/console-feed/pull/105

**Comment by menewman** [comment]
Hurray for open-source! That's great news 🙂
